website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md

```diff
@@ -1,38 +1,83 @@
 ---
 sidebar_position: 3
+toc_max_heading_level: 4
 ---
-# Custom MongoDB & Redis
+# External MongoDB and Redis
 
-Appsmith runs it's own instances of MongoDB database for storing all application data, and Redis for storing session information. It's easy to switch to externally running instances of these services, by just changing the relevant environment variables.
+Appsmith runs embedded instances of MongoDB and Redis. Appsmith uses MongoDB and Redis for data storage and session management. You can use external instances by configuring the instance. External instances are stateless and help in better performance and scalability. Follow the steps on this page to set up external MongoDB and Redis for Appsmith.
 
-:::info Important
-If you are currently using an older version of MongoDB, it's important to upgrade to version 5.0 before connecting Appsmith to an external MongoDB instance. For more information, follow the steps detailed on MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
+:::caution Attention
+To use a custom MongoDB with Appsmith v1.9.0 and later, you need MongoDB version 5 or higher. Follow the steps detailed in the MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
```

Comment:
While both "on later" and "onwards" are used to indicate continuing from a certain point, "onwards" is more commonly used in British English. Since we are adhering to the rules and conventions of American English, we should use "on later" instead of "onwards." Also, technically `later` or `higher` terms signify that the compatibility is with v5 and versions that are released after v5. Let's discuss this if needed.

Reply:
done.